bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/SPTargetPio.java

```diff
@@ -46,7 +46,7 @@
     private static final String _PERIHELION = "perihelion";
     private static final String _EPOCH_OF_PERIHELION = "epochOfPeri";
 
-    private static final DateFormat formatter = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.FULL);
+    public static final DateFormat formatter = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.FULL);
```

Comment:
(Made this public so I could test `Date` equality in terms of the PIO representation.)